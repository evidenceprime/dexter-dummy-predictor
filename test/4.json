{
  "text": "Some fragments of testis and cauda epididymis were fixed by immersion in 2.5% glutaraldehyde for 2 hours, postfixed with osmium tetroxide, and routinely prepared for transmission electron microscopy studies.", 
  "sections": [
  ]
}
