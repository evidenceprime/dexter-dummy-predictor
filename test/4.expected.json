{
  "mentions": [
    {
      "id": 1,
      "tag": "PERCENT",
      "offsets": [
        73,
        76
      ],
      "words": [
        "2.5",
        "%"
      ]
    },
    {
      "id": 3,
      "tag": "TIME",
      "offsets": [
        97,
        99
      ],
      "words": [
        "2",
        "hours"
      ]
    }
  ],
  "equivalences": [
    {
      "id": 0,
      "tag": "PERCENT",
      "items": [
        {
          "ids": [
            1
          ],
          "text": "2.5 %"
        }
      ]
    },
    {
      "id": 1,
      "tag": "TIME",
      "items": [
        {
          "ids": [
            3
          ],
          "text": "2 hours"
        }
      ]
    }
  ]
}